ArduPlane/Parameters.cpp

```diff
@@ -1086,6 +1086,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Bitmask: 11: Disable suppression of fixed wing rate gains in ground mode
     // @Bitmask: 12: Enable FBWB style loiter altitude control
     // @Bitmask: 13: Indicate takeoff waiting for neutral rudder with flight control surfaces
+    // @Bitmask: 14: Climb the aircraft without doing slope in AUTO. If enabled, aircraft will climb with maximum TECS performance. If disabled, aircraft will gradually increase altitude so that altitude and final location are reached at the same time.
```

Comment:
I don't think you can use commas here. It confuses the metadata parser.

Reply:
The comment has been changed

Reply:
I made the changes as suggested